modules/database/funcs.go

```diff
@@ -45,6 +45,27 @@ func NewDatabase(
 		databaseUser: databaseUser,
 		secret:       secret,
 		labels:       labels,
+		name:         "",
```

Comment:
I don't think we want to have a different name and databaseName for a MariaDBDatabase.

Reply:
Hm. I might need to reconsider this.  The k8s Name field needs to be in a format of a subdomain so the charachter set is restricted. I.e. you cannot use underscore in the Name. But DB schema names can have underscores and nova heavily uses it by default e.g. nova_cell0. So we might really need both name and databaseName in the Database structure.